test/e2e/tests/request-queuing/switchChain-watchAsset.spec.js

```diff
@@ -32,65 +32,28 @@ describe('Request Queue SwitchChain -> WatchAsset', function () {
             },
           ],
         },
+        smartContract,
```

Comment:
We add the following in order to skip most of the initial setup steps for the test:
- .withPermissionControllerConnectedToTestDapp()
- smartContract,